ghost/admin/app/components/modal-tier.hbs

```diff
@@ -305,22 +250,18 @@
                                                 </span>
                                             {{/if}}
                                         {{/if}}
-                                    {{else}}
```

Comment:
I think the removed `{{else}}` should be the one above? This one is the opposite of the `{{#if this.stripeYearlyAmount}}` line.

Sidenote: The improper indenting here makes the conditionals harder to follow 😓